src/index.html

```diff
@@ -22,6 +22,399 @@
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              data-qa="nav-hover"
+              href="index.html"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="card__stars">
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+          </div>
+
+          <p class="card__reviews-p">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
```

Comment:
The `href` attribute for a link that is meant to be a button should not be `#`. If this button is supposed to submit a form or perform an action without navigating, consider using a `<button>` element instead of an `<a>` element.